Ports/.port_include.sh

```diff
@@ -122,16 +134,20 @@ fetch() {
 
     # check signature
     if [ "$auth_type" == "sig" ]; then
-        if $(gpg --verify $auth_opts); then
-            echo "- Signature check OK."
+        if which gpg; then
+            if $(gpg --verify $auth_opts); then
+                echo "- Signature check OK."
+            else
+                echo "- Signature check NOT OK"
+                for f in $files; do
+                    rm -f $f
+                done
+                rm -rf "$workdir"
+                echo "  Signature mismatching, removed erronous download. Please run script again."
+                exit 1
+            fi
         else
-            echo "- Signature check NOT OK"
-            for f in $files; do
-                rm -f $f
-            done
-            rm -rf "$workdir"
-            echo "  Signature mismatching, removed erronous download. Please run script again."
-            exit 1
+            echo "gpg not installed; skipping signature validation."
```

Comment:
oof

Reply:
Whats the complaint here? I think it's reasonable to not check signatures if the user does not have `gpg`.

Reply:
This change effects ports both inside Serenity and on the build host.

This change has security implications and isn't mentioned anywhere in the PR description or commits.

For all other other auth types (md5, sha1, etc) a failed signature match is considered a fatal error and the installation is abandoned.

Reply:
I'm not sure the best approach here if we want to install ports from inside Serenity. Serenity doesn't have a `gpg` port yet. And even if it did, it would likely be a cyclic dependency requiring gpg to verify the key to install gpg.

When building ports on the build host, failed signature validation due to missing `gpg` should be fatal.

When building ports on the Serenity host, print a warning due to missing `gpg`. Although I'm generally not keen on undermining package validation on an adhoc basis with a single line warning.

Reply:
what do you sugguest I do about this. I can remove this, or I can add a `noverify` flag to `./package.sh` so it can be used in serenity with some caution.

Reply:
> 
> 
> what do you suggest I do about this? I can remove this, or I can add a `noverify` flag to `./package.sh` so it can be used in serenity with some caution.

Something akin to an optional `noverify` flag would likely be the best long term solution.

Reply:
i've added the `--no-gpg-verification` flag